src/main/java/com/productdock/library/search/adapter/out/elastic/BookElasticRepository.java

```diff
@@ -34,6 +34,11 @@ public void save(Book book) {
         elasticRepository.save(bookDocument);
     }
 
+    @Override
+    public void deleteById(String bookId) {
+        elasticRepository.deleteById(bookId);
```

Comment:
Can this method throw exception, if yes, how it will affect kafka, and how can we recover from that?